ldevents.c

```diff
@@ -102,6 +102,8 @@ addNodeToJSONObject(cJSON *obj, const char *key, LDNode *node)
     case LDNodeHash:
         cJSON_AddItemToObject(obj, key, LDi_hashtojson(node->h));
         break;
```

Comment:
I think it is also possible in this context for the type to be `LDNodeArray`, so we have some missing logic here. However, we can fix that separately. The `default` is still correct in any case.

Reply:
Actually, let's fix it now, if you don't mind.

```
case LDNodeArray:
    cJSON_AddItemToObject(obj, key, LDi_arraytojson(node->a));
```

Reply:
Done